drivers/dali/timings.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2025 by Sven Hädrich <sven.haedrich@sevenlab.de>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef DRIVERS_DALI_INCLUDE_DALI_STD_H_
+#define DRIVERS_DALI_INCLUDE_DALI_STD_H_
```

Comment:
```suggestion
#ifndef DRIVERS_DALI_TIMINGS_H_
#define DRIVERS_DALI_TIMINGS_H_
```

Same for closing comment